src/pages/Swap/index.tsx

```diff
@@ -345,7 +345,7 @@ export default function Swap({ history }: RouteComponentProps) {
     [onCurrencySelection]
   )
 
-  const swapIsUnsupported = useIsSwapUnsupported(currencies?.INPUT, currencies?.OUTPUT)
+  const swapIsUnsupported = useIsSwapUnsupported(currencies[Field.INPUT], currencies[Field.OUTPUT])
```

Comment:
any idea why we werent doing this before?

Reply:
no clue, but it's done this way earlier in the file. probably just multiple editors over time.